db.go

```diff
@@ -54,7 +54,10 @@ var DefaultOptions = &Options{
 
 // Options of the DB storage.
 type Options struct {
-	// Segments (wal files) max size
+	// Segments (wal files) max size.
+	// WALSegmentSize = 0, segment size is default size.
```

Comment:
Right now it takes the default size for `WALSegmentSize < -1` too. Should we make it disabled for `WALSegmentSize < 0` then?

Reply:
yep , not a bad idea. I like it.